sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -1904,28 +1960,72 @@
                                 "minMode": "day",
                                 "maxMode": "year",
                                 "yearRows": 4,
-                                "yearColumns": 5,
-                                "minDate": null,
-                                "maxDate": null
+                                "yearColumns": 5
                               },
                               "enableMaxDateInput": false,
                               "enableTime": false,
+                              "timePicker": {
+                                "hourStep": 1,
+                                "minuteStep": 1,
+                                "showMeridian": true,
+                                "readonlyInput": false,
+                                "mousewheel": true,
+                                "arrowkeys": true
+                              },
+                              "multiple": false,
+                              "defaultValue": "",
+                              "defaultDate": "",
                               "customOptions": {
                                 "dateFormat": "yyyy-MM-dd"
                               },
+                              "persistent": true,
+                              "protected": false,
+                              "dbIndex": false,
+                              "encrypted": false,
+                              "redrawOn": "",
+                              "clearOnHide": true,
+                              "customDefaultValue": "",
+                              "calculateValue": "console.log(data.studyStartDate);\nconsole.log(data.studyEndDate);\nconsole.log(value);\nvalue = value;",

```

Comment:
@ann-aot I was working on this form and there are probably some changes that does not belong to your PR but they are not major. Since you are adding this to the git, can you please just remove the `console.log`.